demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -31,16 +33,21 @@ def create_content_graph(
         dependencies (bool): Whether to create the dependencies.
         output_path (Path): The path to export the graph zip to.
     """
-    ContentGraphBuilder(content_graph_interface).create_graph()
-    if dependencies:
-        content_graph_interface.create_pack_dependencies()
-    if output_path:
-        output_path = output_path / marketplace.value
-    content_graph_interface.export_graph(output_path)
-    logger.info(
-        f"Successfully created the content graph. UI representation is available at {NEO4J_DATABASE_HTTP} "
-        f"(username: {NEO4J_USERNAME}, password: {NEO4J_PASSWORD})"
-    )
+    for _ in range(MAX_RETRIES):
```

Comment:
I prefer not to touch the functions themselves, but add a decorator to them, e.g.:

```
@recover_if_fails
def create_content_graph(
...
```
then (based on `recover_on_error()`):
```
def recover_if_fails():
    def func_wrapper(*args, **kwargs):
        try:
            func(*args, **kwargs)
        except Exception:
            if not neo4j_service.is_running_on_docker():
                logger.error(
                    "Something is broken in the neo4j local configuration, check `neo4j.conf` and `apoc.conf` files.",
                    exc_info=True,
                )
                raise
            logger.warning(
                "Failed to build content graph, retrying with a clean environment.",
                exc_info=True,
            )
            neo4j_service.stop(force=True, clean=True)
            neo4j_service.start()
            func(*args, **kwargs)
        return func_wrapper
```